core/src/main/resources/scripts/base/rdbms/db-grants.sql

```diff
@@ -11,6 +11,10 @@ Grant all privileges on database klaw to kafkauser;
 
 /* Mysql */
 
```

Comment:
Is this a separate issue?

Reply:
Usually for mysql, I had to set this sometimes. But a db admin would know better.